src/middlewared/middlewared/alembic/versions/12.0/2021-12-22_12-59_add_server_tls_uri_to_s3_config.py

```diff
@@ -7,13 +7,29 @@
 import sqlalchemy as sa
 from OpenSSL import crypto
 
+from middlewared.validators import Hostname
```

Comment:
We should avoid importing from middleware in migrations, `Hostname` should be regex iirc, we can just use it directly

Reply:
May I know why should we not import a util from `middlewared` package? 🤔

Reply:
http://docs.middleware.ixsystems.net/database/migrations.html#migrations-tips